code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -15,6 +15,14 @@
 	heatmod = 0.5 // = 1/4x heat damage
 	burnmod = 0.5 // = 1/2x generic burn damage
 
+/datum/species/jelly/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	if(chem.id == "sodiumchloride")
+		H.adjustFireLoss(5)

```

Comment:
This is silly, 0.1 will deal the exact same amount of damage as 10.

Reply:
Wouldn't that issue apply to almost any toxin that you can add an amount less than its absorption rate, though?